src/main/resources/view/MainWindow.fxml

```diff
@@ -46,6 +47,19 @@
           </padding>
         </StackPane>
 
+        <HBox id="contactList" fx:id="contactListPane" styleClass="pane-with-border" minWidth="340" prefWidth="340">
+            <!-- Contact list -->
+            <VBox fx:id="personList" styleClass="pane-with-border" VBox.vgrow="ALWAYS" minWidth= "400" prefWidth="400">
+                <padding>
+                <Insets top="10" right="10" bottom="10" left="10" />
+                </padding>
+                <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
```

Comment:
```suggestion
              <StackPane fx:id="personListPanelPlaceholder" />
```
Indentation, add space before self-closing bracket

`VBox.vgrow` should be removed since the enclosing container is not a VBox anymore.